packages/docgen/lib/get-export-entries.js

```diff
@@ -31,7 +26,7 @@ module.exports = ( token ) => {
 					name = t.declaration.left.name;
 					break;
 				default:
-					name = get( t.declaration, [ 'id', 'name' ], '*default*' );
+					name = t.declaration.id?.name ?? '*default*';
```

Comment:
Adding a note that `?.` + `??` aren't strictly equivalent to `get()`, in that the former check for a value being nullish (`null` or `undefined`), whereas the latter checks for it being `undefined` only.

That said, this looks to still do the right thing in this case, as we wouldn't expect for `null` to be a correct value.

Reply:
I confirm this is the intention. IIUC we can expect `undefined` but not `null` there.